packages/api2/src/_shared/shared.module.ts

```diff
@@ -0,0 +1,7 @@
+import { Module } from '@nestjs/common';
+import { SharedService } from './shared.service';
+
+@Module({
+  providers: [SharedService],
+})
+export class SharedModule {}
```

Comment:
What is the purpose of the shared module?

Reply:
Shared module contains utils, pipes etc that can’t be in a separate module but are needed in multiple modules